Dizzy/Screens/Discovery/PlacesInteractor.swift

```diff
@@ -66,4 +78,23 @@ class PlacesInteractor: PlacesInteractorType {
             }
         }
     }
+
+    func increment(analyticsType: AdminAnalyticsType, by count: Int = 1, to place: PlaceInfo) {
+        let currentCount: Int
+        
+        switch analyticsType {
```

Comment:
i have an idea to remove this switch case because if we will have 50 typse this switch will have 50 cases.
i think it will be better to make the enam get `placeInfo` in it's constructor and each type will hold it's relevant value as associate value  
and then in this function you only need to write `let currentCount = analyticsType.value` or something like that